### PR TITLE
[JENKINS-60995] Fix downstream build link

### PR DESCRIPTION
**Pull request by Vlatombe**
Downstream of https://github.com/jenkinsci/pipeline-build-step-plugin/pull/54

**Work** in progress
* [x] Add an ATH test reproducing the issue
* [x] Link with incrementals build from https://github.com/jenkinsci/pipeline-build-step-plugin/pull/54

# Description

See [JENKINS-60995](https://issues.jenkins-ci.org/browse/JENKINS-60995).

# Submitter checklist
- [ ] Link to JIRA ticket in description, if appropriate.
- [ ] Change is code complete and matches issue description
- [ ] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -273,6 +273,12 @@
             <type>pom</type>
         </dependency>
 
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>pipeline-build-step</artifactId>
+            <version>2.14-SNAPSHOT</version>
```
Comment (bitwiseman) [inline pom.xml:279]:
```suggestion
            <version>2.14-rc430.69345dedbb03</version>
```

**Comment by Vlatombe** [comment]
@olamy Unfortunately I'm not able to run the ATH properly in local (the browser is unable to each the Jenkins instance through `127.0.0.1:<port>`, even though it seems to work for api calls).

**Comment by olamy** [comment]
@Vlatombe no worries. https://github.com/jenkinsci/blueocean-plugin/tree/master/acceptance-tests 
Very important is `~/.blueocean-ath-config` 
Especially 
```
webDriverType=chrome
webDriverUrl=http://localhost:9515
```
And you need a `chromedriver` running.

because of Docker network, you should run: `./run.sh -ld`

**Comment by olamy** [comment]
@Vlatombe  Still a draft? sounds good to merge.

**Comment by Vlatombe** [comment]
> @Vlatombe Still a draft? sounds good to merge.

https://github.com/jenkinsci/pipeline-build-step-plugin/pull/54 needs a release

File: pom.xml [hunk 2]
```diff
@@ -273,6 +273,12 @@
             <type>pom</type>
         </dependency>
 
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>pipeline-build-step</artifactId>
+            <version>2.14-rc430.69345dedbb03</version>
```
Comment (Vlatombe) [inline pom.xml:279]: Pending release of https://github.com/jenkinsci/pipeline-build-step-plugin/pull/54
```suggestion
            <version>2.14</version>
```

**Comment by olamy** [comment]
argh yep we need release of that.
I can't but did that https://github.com/jenkins-infra/repository-permissions-updater/pull/2015

**Comment by olamy** [comment]
@Vlatombe finally pipeline-build-step-plugin has been released.
I moved this PR to ready to review ;)

**Pull request merged**